### DISCUSSION TITLE
Creating a new outline via link menu/dialog

### DISCUSSION THREAD

**Discussion by latvia234** [vincode-io/Zavala#120] [Ideas]
Is it already possible to create an outline via the link screen/dialog? (Don't know how you call the screen that you can see below.) Could not find it in the iOS app. Maybe it exists already, but I don't know.

When typing in the name of the link, it would be helpful if there was a menu option available to create a new outline from here. When typing the name of the link, the name would also be visible in the first menu option right after "Create Outline: [name link]".

I created this example menu:

![createfromlinkdialog](https://user-images.githubusercontent.com/86680397/131221175-9ec5001b-f46e-4e70-a34f-5c0ee8adccc2.png)

**Comment by vincode-io** [comment]
Sorry, that feature is coming in the 1.1 release.

**Reply by vincode-io** [reply]
The 1.1 release is out now. You will see a "+" in the navigation bar that will create outlines for you.

**Reply by latvia234** [reply]
Thanks for creating this! It works right now on iOS 14.7.1 with Zavala 1.1

For other people who are interested in Zavala I made a video that you can see below.

One small detail though, I think you already did notice it - but in the case that you did not notice it: If I tab on the link at 00:30 in the video, the outline is not visible.


https://user-images.githubusercontent.com/86680397/131243691-c8dfbf87-e5d2-46c4-8d32-0761a9af3595.mov

**Reply by vincode-io** [reply]
Thanks for the heads up. I wasn't aware of that. I'll look into it first thing.

**Reply by vincode-io** [reply]
That was a hard one to track down. It only happens right after a new app launch and only on the iPhone. Anyway, it is fixed now and will be in the upcoming 1.1.1 release sometime in the next week or so.

**Reply by latvia234** [reply]
I have no experience as a developer but I can imagine that some bugs are harder to track down.

It was a small detail but important enough to fix... Thanks for the 1.1.1 fix!
